### PR TITLE
defaultStore option added

### LINKED ISSUE #392: Allow to change defaultStore in createAtom
We need an option `defaultStore` in [AtomOptions](https://github.com/artalar/reatom/blob/v2/packages/core/src/createAtom.ts#L46). With this a user may create own createAtom implementation with own store, which useful for its customization:

```ts
// utils/store.ts
export const store = createStore({
  onPatch: console.log
})

// utils/createAtom.ts
import {createAtom as createAtomOriginal} from '@reatom/core'
export const createAtom: typeof createAtomOriginal = (deps, reducer, options) =>
  createAtomOriginal(deps, reducer, {...options, defaultStore: store})
```

### PR DESCRIPTION
**Pull request by Akiyamka**
closes #392

### CODE REVIEW

File: packages/core/README.md [hunk 1]
```diff
@@ -451,7 +451,13 @@ const formAtom = createAtom(
 
 > `Outdated track call` is throwed when you try to use reactive handlers outside synchronous reducer call. For example, `schedule` is called only after all atoms recalculation.
 
-> TODO: `options`
+The third argument `options` allows you to override the default atom settings:
+id - setup id for this atom
+decorators - array of (atom decorators)[#atom-decorators] 
+defaultStore - redefine store for this atom
```
Comment (artalar) [inline packages/core/README.md]:
```suggestion
defaultStore - redefine store bindings for `dispatch`, `getState` and `subscribe` of this atom
```

File: packages/core/src/createAtom.ts [hunk 2]
```diff
@@ -9,16 +9,16 @@ import {
   Cause,
   createReatomError,
   createTemplateCache,
-  defaultStore,
+  defaultStore as originalStore,
```
Comment (artalar) [inline packages/core/src/createAtom.ts]:
```suggestion
  defaultStore as originalDefaultStore,
```

Comment (artalar) [reply packages/core/src/createAtom.ts]: Also, if you can, rename please `callSafetyDefault` to `originalCallSafety` and `createTemplateCacheDefault` to `originalCreateTemplateCache` [here](https://github.com/artalar/reatom/blob/v2/packages/core/src/createStore.ts)

**Review by artalar — CHANGES_REQUESTED** [review]

File: packages/core/src/createAtom.ts (hunk 2 above)
Comment (Akiyamka) [reply packages/core/src/createAtom.ts]: done

**Comment by artalar** [comment]
@allcontributors please add @Akiyamka for code

**Pull request merged**